WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -434,8 +434,8 @@ private extension OrdersViewController {
 //
 private extension OrdersViewController {
 
-    func lookUpOrderStatus(for order: Order) -> OrderStatus? {
-        for orderStatus in currentSiteStatuses where orderStatus.slug == order.statusKey {
+    func lookUpOrderStatus(for order: Order?) -> OrderStatus? {
+        for orderStatus in currentSiteStatuses where orderStatus.slug == order?.statusKey {
```

Comment:
nit: might it make sense to early return `nil`? otherwise it might return some order status that happens to have an empty slug 🤔

Reply:
Good catch! Fixed in 6b2b5e8626f799bdb8db4a3ce20107fe1b7d8330.